Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue8529_1.xaml.cs

```diff
@@ -0,0 +1,174 @@
+﻿using Xamarin.Forms.Internals;
+using Xamarin.Forms.Xaml;
+using System;
+using System.Windows.Input;
+using System.Threading.Tasks;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if APP
+	[XamlCompilation(XamlCompilationOptions.Compile)]
+#endif
+	public partial class Issue8529_1 : ContentPage
+	{
+		public Issue8529_1()
+		{
+#if APP
+			InitializeComponent();
+#endif
+			BindingContext = new Issue8529ViewModel();
+		}
+	}
+
+	[Preserve(AllMembers = true)]
+	public class Issue8529ViewModel
```

Comment:
Can you  make these classes children of *Issue8529_1* ?  Just so we don't add a bunch of extra classes to the *Xamarin.Forms.Controls.Issues* namespace

This PR looks great!! Thank you